smart-contracts.asciidoc

```diff
@@ -130,52 +131,59 @@ To accomplish this the contract must impose two access restrictions:
 
 Solidity function modifers offer a concise way to implement these restrictions.
 
+_Note: The following example uses an underscore semicolon within the modifier bodies. This is  a Solidity feature used to tell the compiler when to run the modifed function's body. A developer can act as if the modifed function's body will be copied to the position of the underscore._
```

Comment:
For **clarification**, here I'd add something like "_The code for the function being modified is inserted where the _ is placed in the modifier_".

Reply:
Yes, that explanation is better. Does the compiler user a "jump" or does it inline a complete copy of the code into _?